Dockerfiles/dogstatsd/socat-proxy/README.md

```diff
@@ -24,7 +24,7 @@ spec:
       containers:
         - **[insert producer container specs]**
         - name: socat
-          image: datadog/dogstatsd-socat-proxy:beta
+          image: datadog/dogstatsd-socat-proxy:latest
```

Comment:
It seems there is no `latest` tag for dogstatsd-socat-proxy : https://hub.docker.com/r/datadog/dogstatsd-socat-proxy/tags/

Reply:
Good catch. I changed that back to beta for now.